pandas/tests/indexes/period/test_period.py

```diff
@@ -570,3 +572,20 @@ def test_maybe_convert_timedelta():
     offset = offsets.BusinessDay()
     with pytest.raises(ValueError, match='freq'):
         pi._maybe_convert_timedelta(offset)
+
```

Comment:
can you put here (create a new file)
``pandas/tests/series/indexing/test_mult.py``